### ISSUE TITLE
Add HSTS bypass by man-in-the-middile'ing NTP

### ISSUE THREAD

**Issue by byt3bl33d3r** [byt3bl33d3r/MITMf#39]
as described here https://www.blackhat.com/docs/eu-14/materials/eu-14-Selvi-Bypassing-HTTP-Strict-Transport-Security-wp.pdf

And PoC code here https://github.com/PentesterES/Delorean

**Comment by pomwtin** [comment]
Another demo, same protocol, other tool:
youtube.com/watch?v=m-o-UPBLqvU

**Comment by byt3bl33d3r** [comment]
@pomwtin heh that's actually an interesting take on it.. That's more of a social engineering attack at that point though

**Comment by pomwtin** [comment]
True, but i didnt find any other way of doing this on default windows settings.
Auto-sync allows just a slight offset to be corrected, timeslide is therefor not possible- windows update time once a week.

However, on some linux and mac distros timeslide is possible.

**Comment by WiredLife** [comment]
works great with mitmf
only do
iptables -t nat -A PREROUTING -p udp --dport 123 -j REDIRECT --to-port 123
and run ./delorean.py in the delorean dir
